libs/perun/services/src/lib/auth.service.ts

```diff
@@ -144,51 +126,25 @@ export class AuthService {
   }
 
   logout() {
-    this.manager.signoutRedirect()
-      .catch(function(error) {
-        console.log(error);
-      });
+    this.oauthService.revokeTokenAndLogout().catch(function(error) {
+      console.log(error);
+    });
```

Comment:
There are some problems with logout - application throws an error on revoking token, so the logout is broken.